.github/workflows/deploy-production.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy Production
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  back-merge:
+    needs: deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Merge master -> develop
+        uses: devmasx/merge-branch@v1.3.1
+        with:
+          type: now
+          from_branch: master
+          target_branch: develop
+          github_token: ${{ secrets.GH_TOKEN }}
+  deploy: # Runs install once before everything to create & save cache
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions/setup-node@v2.4.0
+        with:
+          node-version: '14'
+          cache: 'npm'
```

Comment:
Do we need to run `npm` instead of `yarn` for this repo?

Reply:
yes